package.json

```diff
@@ -14,7 +14,7 @@
   "homepage": "http://videojs.com",
   "author": "Steve Heffernan",
   "scripts": {
-    "test": "grunt test && if [ '$TRAVIS' ]; then grunt coveralls; fi;"
```

Comment:
you could also just double quote and escape travis here like `grunt test && if [ \"$TRAVIS\" ]; then grunt coveralls; fi;"` but if were going for windows build compatibility then grunt is he better way. LGTM other than that